libs/libc/signal/sig_delset.c

```diff
@@ -62,13 +62,12 @@
  *
  ****************************************************************************/
 
-int sigdelset(FAR sigset_t *set, int signo)
+int nxsig_delset(FAR sigset_t *set, int signo)
 {
   /* Verify the signal */
 
   if (!GOOD_SIGNO(signo))
     {
-      set_errno(EINVAL);
       return ERROR;
```

Comment:
Similar to the above.  ERROR should never be returned inside the OS.  It should return -EINVAL if the signal number is invalid.  There are probably updates need to the Return Value comments above too.

Reply:
Done.